src/platform_impl/apple/appkit/view.rs

```diff
@@ -639,19 +639,32 @@ declare_class!(
         }
 
         #[method(mouseEntered:)]
-        fn mouse_entered(&self, _event: &NSEvent) {
+        fn mouse_entered(&self, event: &NSEvent) {
             trace_scope!("mouseEntered:");
-            self.queue_event(WindowEvent::CursorEntered {
+
+            let Some(position) = self.mouse_view_point(event) else {
+                return
+            };
```

Comment:
@madsmtm I couldn't find in the [documentation](https://developer.apple.com/documentation/appkit/nsevent/1529068-locationinwindow) if this works.

Same for `mouseExited`.

Reply:
The documentation says that it works for all "mouse events":
> This property applies to mouse events

This includes `mouseEntered:` and `mouseExited:` (I've tested that it works, and indeed it does).